Project.toml

```diff
@@ -56,7 +56,7 @@ Tables = "0.2, 1"
 TypedTables = "1.2"
 Unitful = "0.18, 1"
 YAML = "0.3, 0.4"
-julia = "1"
+julia = "1.6"
```

Comment:
Is this the minimum required? Or should it be `"1.6,1"`?

Reply:
"1.6" translates to [1.6, 2[